.gitignore

```
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+.env
```

Línea en blanco faltante.
```suggestion
.env

```

---

👍